data/Garvitaj12.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Garvita Jain",
+  "type": "personal",
+  "bio": "B.Tech student",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
```

Comment:
The links & icon are for Github, meanwhile you have written "Follow me on Twitter"

Reply:
> The links & icon are for Github, meanwhile you have written "Follow me on Twitter"

Given the length of time this PR has been open and as this is not a blocker, I will merge this.  If you wish to make changes to your Profile then I suggest you have a look at our Editing Guide in Docs: https://linkfree.io/docs/how-to-guides/editing